vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/RowDesc.java

```diff
@@ -26,16 +31,20 @@
 public class RowDesc {
 
   private final List<String> columnNames;
+  private final Map<String, Integer> columns; // key - column name, value - column index (if multiple columns with same names then last)
 
   public RowDesc(List<String> columnNames) {
     this.columnNames = columnNames;
+    this.columns = Collections.unmodifiableMap(IntStream.range(0, columnNames.size())
```

Comment:
please simply create a `HashMap` that is modifiable (as it remains interna) and then a simple loop to put stuff in the map without lambdas and collectors